### PR TITLE
Add learning objectives

### LINKED ISSUE #7: Add Learning Objectives
in `index.Rmd`

### PR DESCRIPTION
**Pull request by KatherineCox**
closes #7

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -27,10 +30,92 @@ _Programming skills_
...
+- Explain what vectors and dataframes are in R
+- Use common commands to manipulate R objects (e.g. assigning, printing, subsetting, mathematical operations)
+- Carry out a basic exploration of rectangular data using base R commands (e.g. `dim`, `summary`, `str`)
 
-<!-- - Objective 1 -->
-<!-- - Objective 2 -->
-<!-- - Objective 3 -->
+## Core Competencies {-}
+
+This activity addresses the following core concepts and competencies:
```
Comment (cutsort) [inline index.Rmd:41]: These seem appropriate but take a bit of thought to map onto the exercises.  Beyond the scope of this PR but perhaps worth adding a third column with some breadcrumbs e.g. us_contagious_diseases address V&C #2 and #4 while swirl addresses NIBLSE C7

File: index.Rmd [hunk 2]
```diff
@@ -10,10 +10,13 @@ description: "This book provides a lab exercise that introduces students to the
 favicon: assets/GDSCN_style/gdscn_favicon.ico
 ---
 
-# About this Course {-}
+# About {-}
```
Comment (cutsort) [inline index.Rmd:13]: Note that this exercise fits well into a 3 hour lab (and that the Instructor Guide summarizes how that time budget is spent across the activities/exercises)

File: index.Rmd [hunk 3]
```diff
@@ -27,10 +30,92 @@ _Programming skills_
 
 ## Learning Objectives {-}
 
-<!-- Learning objectives for this activity come from the [Genetics Core Competencies](https://genetics-gsa.org/education/genetics-learning-framework/): -->
+- Open RStudio and locate important panels (R console, Help)
+- Run commands in the R console
+- Explain what vectors and dataframes are in R
+- Use common commands to manipulate R objects (e.g. assigning, printing, subsetting, mathematical operations)
+- Carry out a basic exploration of rectangular data using base R commands (e.g. `dim`, `summary`, `str`)
```
Comment (cutsort) [inline index.Rmd:37]: How about structuring to a short phrase with details in parentheses? e.g.
- Open RStudio on AnVIL (e.g. set up account, locate R console and Help panels)
- Practice working in R Console (e.g. run commands)
- Use swirl to learn basic R commands (e.g. assign, print, subset, math operations)
- Explore a dataset using base R commands (e.g. `dim`, `summary`, `str`)

**Review by cutsort — APPROVED** [review]

File: index.Rmd (hunk 2 above)
Comment (KatherineCox) [reply index.Rmd:13]: @cutsort added a new "Format" section - what do you think?

**Pull request merged**